### ISSUE TITLE
How did you deploy it to vercel?

### ISSUE THREAD

**Issue by liorFaigenzon** [zenstackhq/sample-todo-trpc#2]
I use this :
![image](https://github.com/zenstackhq/sample-todo-trpc/assets/14123246/a67cb599-fcd7-4c32-a5ba-56920b01a3c9)
and get this :
![image](https://github.com/zenstackhq/sample-todo-trpc/assets/14123246/1a019756-15de-4a27-865c-279d24fb7c57)

**Comment by dBianchii** [comment]
I don't think you need to change any settings after you send the code to vercel.
Definitely don't need to change the install script nor the build
